engine/lib/sessions.php

```diff
@@ -76,32 +75,12 @@ function elgg_is_admin_user($user_guid) {
 	}
 
 	// cannot use magic metadata here because of recursion
-
-	// must support the old way of getting admin from metadata
-	// in order to run the upgrade to move it into the users table.
-	$version = (int) elgg_get_config('version');
-	
-	if ($version < 2010040201) {
```

Comment:
Is it taken care of in an upgrade script? Would be nice to keep for easier upgrades